### PR TITLE
feature/OE-268-spreadsheet-view

### PR DESCRIPTION
**Pull request by devsquad-mathews-lima**
## Review

For the submitter, initial self-review:

- [ ] Reviewed the test cases added for completeness and possible points for discussion
- [ ] Check the scope of the changeset - is everything in the diff required for the pull request?

For the pull request reviewer(s), this changeset has been reviewed for:

- [ ] Consistency between the changeset and the goal stated above
- [ ] Usage friction - is the proposed change in usage cumbersome or complicated?
- [ ] Complexity - is the proposed change introducing any unjustified complexity? 
- [ ] Thoroughness of added tests and any missing edge cases

### CODE REVIEW

**Review by ron-junior-devsquad — COMMENTED** [review]
if you have any question tell me, and if all comments in this review are not relevant. ignor them.

File: resources/views/livewire/number-tracker/spreadsheet.blade.php [hunk 1]
```diff
@@ -0,0 +1,550 @@
...
+                                                <span name="pipe">
+                                                    <input
+                                                        type="text" class="text-center w-10 inline"
+                                                        value=""
+                                                        placeholder="D"
+                                                        x-on:input.debounce.400ms="
+                                                            $wire.attachNewDailyEntry({{ $userKey }}, {{ $user->id }}, '{{ $label }}', 'doors', $event.target.value)
+                                                        "
+                                                    />
+                                                </span>
```
[the remark this answers is not in the document]
Comment (devsquad-mathews-lima) [reply resources/views/livewire/number-tracker/spreadsheet.blade.php]: I wrapped the table inside the `x-form` component and moves the store/update to an Action.

I kept with store/update in separated flows. When is adding new daily numbers, I change the code to make a massive insert, and when already exists, just update.

Since the massive insert is a **bulk insert**, it will make just one query to insert, instead of one per user+day.

Basically, if all members don't have any records in `daily_numbers` table, the code will make a massive insert for all members.

After that, the code will only update.

File: app/Http/Livewire/NumberTracker/Spreadsheet.php [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+            });
+    }
+
+    public function getUsersProperty()
+    {
+        return User::query()
+            ->where('office_id', $this->selectedOffice)
+            ->whereNotIn('role', ['Admin', 'Owner'])
+            ->with([
+                'dailyNumbers' => function ($query) {
```
[the remark this answers is not in the document]
Comment (devsquad-mathews-lima) [reply app/Http/Livewire/NumberTracker/Spreadsheet.php:117]: after talking with Yuki, Admins and Owners must not be on the table.
